### PR TITLE
chore: data proxy test disconnect and query again

### PR DESCRIPTION
**Pull request by Jolg42**
Errors with
`getDmmf is not yet supported`
https://github.com/prisma/ecosystem-tests/runs/7135336866?check_suite_focus=true#step:6:513

[Related Slack thread](https://prisma-company.slack.com/archives/C016KUHB1R6/p1656588313002569)

### CODE REVIEW

File: dataproxy/nodejs/package.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
     "test": "jest"
   },
   "dependencies": {
-    "@prisma/client": "4.1.0-dev.8"
+    "@prisma/client": "prisma@4.1.0-integration-fix-disconnect-dmmf.1"
```
Comment (janpio) [inline dataproxy/nodejs/package.json]:
```suggestion
    "@prisma/client": "4.1.0-integration-fix-disconnect-dmmf.1"
```

File: dataproxy/nodejs/package.json [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
   },
   "devDependencies": {
     "@types/jest": "28.1.3",
     "@types/node": "16.4.1",
     "jest": "28.1.2",
-    "prisma": "4.1.0-dev.8",
+    "prisma": "prisma@4.1.0-integration-fix-disconnect-dmmf.1",
```
Comment (janpio) [inline dataproxy/nodejs/package.json]:
```suggestion
    "prisma": "4.1.0-integration-fix-disconnect-dmmf.1",
```

File: dataproxy/nodejs/package.json (hunk 1 above)
Comment (aqrln) [reply dataproxy/nodejs/package.json]: 🤦

Comment (aqrln) [reply dataproxy/nodejs/package.json]: thank you!

**Pull request merged**